### PR TITLE
Add Example Docs for DrawerSubheader component

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-5669 .

#### Changes proposed in this Pull Request:

-Use PrismJS to highlight targeted lines in the code example blocks

-Include buttons to copy to clipboard and view full github source

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheader.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <DrawerNavItem title="Locations" itemID="2" />
+        </DrawerNavGroup>
+    </DrawerBody>
+</Drawer>
+`;
+
+export const DrawerSubheader = (): JSX.Element => (
+  <Box>
+    <DrawerSubheaderExample />
+    <CodeBlock code={codeSnippet} language="jsx" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheader.tsx:24]: please add dataLine="4-6"

File: docs/src/componentDocs/DrawerSubheader/examples/InteractiveDrawerSubheader.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            />
+        </DrawerNavGroup>
+    </DrawerBody>
+</Drawer>
+`;
+
+export const InteractiveDrawerSubheader = (): JSX.Element => (
+  <Box>
+    <InteractiveDrawerSubheaderExample />
+    <CodeBlock code={codeSnippet} language="jsx" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/DrawerSubheader/examples/InteractiveDrawerSubheader.tsx:66]: please add dataLine="8-12"

File: docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheaderExample.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  DrawerNavItem,
+  DrawerSubheader,
+} from '@brightlayer-ui/react-native-components';
+import {ExampleShowcase} from '../../../shared';
+import {Text} from 'react-native-paper';
+
+export const DrawerSubheaderExample = (): JSX.Element => (
+  <ExampleShowcase>
+    <Drawer style={{width: 250, margin: 'auto'}}>
+      <DrawerHeader title="Title" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheaderExample.tsx:16]: why header not picking up default theme.colors.surface. Currently its white background

**Comment by JeffGreiner-eaton** [comment]
header title and subHeader text not aligned and header default color not set
image

Shouldn't header default to like 16px padding?

image

**Comment by JeffGreiner-eaton** [comment]
can you update the Drawer SubHeader API table for the divider? The description is wrong and update to

"Optional divider which appears below the Subheader"

File: docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheaderExample.tsx (hunk 3 above)
Comment (manojleaton) [reply docs/src/componentDocs/DrawerSubheader/examples/DrawerSubheaderExample.tsx:16]: As we discussed in the pre-demo meeting, this should get checked/fixed in the RN component library. Thanks

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good with react-native-components": "^8.0.4-beta.2"

**Pull request merged**